packages/ai/src/methods/chrome-adapter.ts

```diff
@@ -85,8 +84,11 @@ export class ChromeAdapter {
       return false;
     }
 
+    const extractedOptions = this.extractCreateOptions(request);
+    const mergedOptions = this.mergeCreateOptions(extractedOptions);
+
     // Triggers out-of-band download so model will eventually become available.
-    const availability = await this.downloadIfAvailable();
+    const availability = await this.downloadIfAvailable(mergedOptions);
 
```

Comment:
I now see the same error in Edge that we used to have in Chrome when we were migrated to the new type: the `LanguageModel.prompt` method exists, but throws an unsupported input error with the new rich type. Edge Canary works. I'll take a look at detecting the Edge version in  isAvailable.